### ISSUE TITLE
Feature request: enable at least one testnet

### ISSUE THREAD

**Issue by PaulRBerg** [antoncoding/opyn-monitor#15]
Before using a defi app with real money, I like playing with it on a testnet, to ensure that the UI is robust enough.

The Opyn Monitor works only on mainnet currently:

<img width="649" alt="Capture d’écran 2020-06-16 à 13 06 18" src="https://user-images.githubusercontent.com/8782666/84761674-6f171c80-afd2-11ea-95f0-2c1c791e7a50.png">

**Comment by antoncoding** [comment]
Yeah I looked into this before and it could be quite a big task given the current infra so it's not gonna happen too soon.. (just a head up). 

I'll try to make this work anyway ;)

**Comment by PaulRBerg** [comment]
Oh, no pressure, I know that Opyn itself doesn't (yet) have a testnet version.

I open GitHub issues because I think they are positive-sum.
